process-compose.yaml

```diff
@@ -38,12 +38,16 @@ processes:
   deploy-sequencer-contracts:
     # The contract addresses are implicitly inherited from .env. We need to unset these or else the
     # script will think they're already deployed.
+    environment:
+      - ESP_TOKEN_NAME="Espresso"
```

Comment:
process compose should pass all values from `.env` to all services already. So I think these shouldn't be needed.

Reply:
thanks, adjusted